metadata.py

```diff
@@ -0,0 +1,64 @@
+"""
+The purpose of this file is to define the metadata of the app with minimal imports. 
+
+DO NOT CHANGE the name of the file
+"""
+
+from mmif import DocumentTypes, AnnotationTypes
+
+from clams.app import ClamsApp
+from clams.appmetadata import AppMetadata
+from lapps.discriminators import Uri
+
+
+# DO NOT CHANGE the function name 
+def appmetadata() -> AppMetadata:
+    """
+    Function to set app-metadata values and return it as an ``AppMetadata`` obj.
+    Read these documentations before changing the code below
+    - https://sdk.clams.ai/appmetadata.html metadata specification. 
+    - https://sdk.clams.ai/autodoc/clams.appmetadata.html python API
+    
+    :return: AppMetadata object holding all necessary information.
+    """
+    
+    # first set up some basic information
+    metadata = AppMetadata(
+        name="Tfidf Keyword Detector",
+        description="",  # briefly describe what the purpose and features of the app
+        app_license="Apache 2.0",  # short name for a software license like MIT, Apache2, GPL, etc.
+        identifier="tfidf-keyword-detector",  # should be a single string without whitespaces. If you don't intent to publish this app to the CLAMS app-directory, please use a full IRI format. 
+        url="https://github.com/clamsproject/app-tfidf-keyword-detector",  # a website where the source code and full documentation of the app is hosted
+        # (if you are on the CLAMS team, this MUST be "https://github.com/clamsproject/app-tfidf-keyword-detector"
+        # (see ``.github/README.md`` file in this directory for the reason)
+        analyzer_version='version_0.0.1', # use this IF THIS APP IS A WRAPPER of an existing computational analysis algorithm
```

Comment:
this app is not a "wrapper" of an off-the-shelf app/model, but something you develop from scratch, so no need to put anything here.